scripts/test/maven/performanceTest.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+#
+# Copyright 2022-2023 Crown Copyright
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+set -e
+
+if [ "$#" -lt 4 ]; then
+  echo "Usage: $0 <test> <shortId> <vpc> <subnets> <optional-maven-params>"
```

Comment:
Looks like this needs to add > 5 to the argument count check to pass in the optional maven args

Reply:
I'm not sure what you mean. It'll display the usage message if there are fewer than 4 arguments. If there are 5 or more it means there were extra Maven parameters passed in. Those will be passed through to deployTest.sh with the `"$@"`, after the `shift 4`.

Reply:
Ah yep ignore me, forgot that the optional params can be any length :smile: